graphql_compiler/compiler/compiler_entities.py

```diff
@@ -142,3 +142,10 @@ def to_gremlin(self):
         The effect of MarkerBlocks is applied during optimization and code generation steps.
         """
         return u''
+
+    def to_cypher(self):
```

Comment:
If we're adding to_cypher() methods, shouldn't we also add an abstract method to_cypher() for CompilerEntity after line 74?

Reply:
Good catch, thank you.

Reply:
Actually, as @pmantica1 points out, we don't output anything other than `Expression` objects this way, for any output target other than Gremlin. So we don't need `to_cypher()` methods on `CompilerEntity` or `MarkerBlock`.